samples/golf/CMakeLists.txt

```diff
@@ -60,24 +57,21 @@ SET(CMAKE_CXX_STANDARD_REQUIRED ON)
 
 SET(OpenGL_GL_PREFERENCE "GLVND")
 
-find_package(CROGINE REQUIRED)
 find_package(SDL2 REQUIRED)
 find_package(OpenGL REQUIRED)
 #TODO specify version 2.83
 find_package(Bullet REQUIRED)
 find_package(SQLite3 REQUIRED)
 
 if(USE_NEWSFEED)
-find_package(CURL REQUIRED)
+  find_package(CURL REQUIRED)
 endif()
 
-if(NOT CROGINE_FOUND)
-  #build it from source
-  add_subdirectory(../../crogine)
```

Comment:
This behaviour simply doesn't work in modern cmake as the crogine folder is not a subdirectory of this golf folder, but there are better modern alternatives like the one I've used - Check if the `crogine` target already exists, and if so we must be being built as a subproject, but if not we must find the pre-installed version